modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.client/src/main/scala/edu/gemini/seqexec/web/client/components/NavBar.scala

```diff
@@ -35,7 +36,7 @@ object NavBar {
               IconTerminal.copy(p = IconTerminal.p.copy(link = true, circular = true, onClick = Callback {SeqexecCircuit.dispatch(ToggleDevConsole)}))
             )
           ),
-          SeqexecCircuit.connect(_.user)(TopMenu(_))
+          userConnect(x => TopMenu(x))
```

Comment:
Perhaps it's the pointfree style bias coming from Haskell but isn't eta reduction a good practice in these cases? Not sure how the syntax would be, would `(TopMenu(_))` work?